ci_tools/bumpversion.sh

```diff
@@ -19,9 +19,7 @@ else
 
```

Comment:
can you fix the spacing/tabs in this file

Reply:
Sure, done.